ui/shared/components/panel/variants/Variants.jsx

```diff
@@ -34,33 +36,63 @@ const StyledVariantRow = styled(Grid.Row)`
   }}
 `
 
-const Variant = ({ variant }) => {
+const StyledCompoundHetLink = styled(Grid.Column)`
+  margin-right: -1em !important;
+`
+
+const StyledCompoundHetRows = styled(Grid)`
+  margin-left: 0em !important;
+  margin-right: 1em !important;
+  margin-top: 0.5em !important;
+  margin-bottom: 0 !important;
+`
+
+const Variant = ({ variant, isCompoundHet }) => {
   const mainGeneId = getVariantMainGeneId(variant)
   return (
-    <StyledVariantRow severity={CLINSIG_SEVERITY[(variant.clinvar.clinicalSignificance || '').toLowerCase()]}>
+    <StyledVariantRow key={variant.variantId} severity={CLINSIG_SEVERITY[(variant.clinvar.clinicalSignificance || '').toLowerCase()]} iscompoundhet={(isCompoundHet || false).toString()} >
```

Comment:
why are you using toString?

Reply:
I got the following error message when using a boolean:

```
backend.js:6 Warning: Received `true` for a non-boolean attribute `iscompoundhet`.

If you want to write it to the DOM, pass a string instead: iscompoundhet="true" or iscompoundhet={value.toString()}.
   ...
```